pages/index.js

```diff
@@ -52,13 +66,40 @@ const StyledNav = styled.nav`
   line-height: 1.2rem;
 `;
 
+const StyledBackButton = styled.input`
+  background-color: transparent;
+  text-decoration: none;
+  color: black;
+  margin: 1rem;
+  padding: 1rem;
+  border-radius: 8.5px;
+  border: 1px solid black;
+  text-align: center;
+  background-color: lightskyblue;
+`;
+
+const StyledMessage = styled.p`
+  align-self: center;
+  text-align: center;
+  font-weight: 600;
+  margin: 1rem auto;
+`;
+
 export default function HomePage({ onAddEmotionEntry }) {
+  const [tension, setTension] = useState("0");
+  const [isShown, setIsShown] = useState(true);
```

Comment:
```suggestion
  const [isFormSubmitted, setIsFormSubmitted] = useState(false);
```

Now I had this idea for a more suitable state name. What do you think?